bundles/org.openhab.core.karaf/src/main/java/org/openhab/core/karaf/internal/FeatureInstaller.java

```diff
@@ -336,15 +339,19 @@ private synchronized void installAddons(final FeaturesService service, final Map
                     Feature[] features = service.listInstalledFeatures();
                     for (String addon : entries) {
                         if (addon != null && !addon.isEmpty()) {
```

Comment:
At the time we use `String.strip()` it might be better to check for `isBlank()` here.

Reply:
It would prevent errors when there is only whitespace between commas: "addon1, ,addon2"

Reply:
Though the whitespace would probably cause the new check `(service.getFeature(id) != null)` to return `null` :wink:

Reply:
But it will still trigger the warning logic:

                                logger.warn("The {} add-on '{}' does not exist - ignoring it.", type, addon.strip());